### ISSUE TITLE
Marketplace: Banner video of paid plugins

### ISSUE THREAD

**Issue by cpapazoglou** [Automattic/wp-calypso#59406]
- [x] find a way to fetch featured video from the API OR use hard coded data
- [x] add banner video in API `marketplace/products`
- [x] render the video in Calypso

**Comment by gcsecsey** [comment]
I've looked into how the video URL is stored on WC.com

So far I’ve found that the `featured_video` field is a meta field on the WC product. As we're already getting the `description` field dynamically from the WC.com REST API, the response could be extended with the `featured_video` field as well.

I've asked the Alpha team whether this is feasible: p1641307725094700-slack-C4E0DVDB2
@andfinally created the issue 12112-gh-Automattic/woocommerce.com to track the changes.

For the time being, I've also started adding the video URLs statically to the public-api response.
